### PR TITLE
completed typescript-forms

### PR DESCRIPTION
**Pull request by mahdiehmaleki**
form-events
form-values

### CODE REVIEW

File: typescript-forms/form-values.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+interface formElements extends HTMLFormControlsCollection {
+  name: HTMLInputElement;
+  email: HTMLInputElement;
+  message: HTMLTextAreaElement;
+}
+
+const $contactForm = document.querySelector<HTMLFormElement>('form');
```
Comment (hesam) [inline typescript-forms/form-values.ts:7]: I'm not familiar with the syntax you're using above: `querySelector<...>`. Does it work?!
The reference answer does it this way: 
`const $contactForm = document.querySelector('#contact-form') as HTMLFormElement;`

File: typescript-forms/javascript-forms-quiz-notes.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+- What event is fired when a user's cursor leaves a form control?
+  blur
+- What event is fired as a user changes the value of a form control?
+  input
+- What event is fired when a user clicks the `"submit"` button within a `<form>`?
+  submit
+- What does the `event.preventDefault()` method do?
+  prevents the default action that belongs to the event from occurring.
+- What does submitting a form without `event.preventDefault()` do?
+  will trigger the form's default action, which typically sends the form data to the server specified in the action attribute.
```
Comment (hesam) [inline typescript-forms/javascript-forms-quiz-notes.md:18]: Right. And if there is no action attribute, the browser will submit the form to itself which will refresh the page and update the url with the form values.

**Review by hesam — APPROVED** [review]
Nice work here 👍 There's a couple of comments in the code. Please see.

File: typescript-forms/form-values.ts (hunk 1 above)
Comment (mahdiehmaleki) [reply typescript-forms/form-values.ts:7]: I guess both ways are correct. <HTMLFormElement> returns an element of the specific type.

**Pull request merged**